adr/ADR-32.md

```diff
@@ -270,7 +285,17 @@ Note the name of the queue group is fixed to `q` and cannot be changed otherwise
 different implementations on different queue groups will respond to the same
 request.
 
-The handler specified by the client to process requests should operate as any
+For each configured endpoint, a queue subscription should be created. Subscription
+subject is composed by concatenating root subject and endpoint name:
+
+```typescript
+{$root_subject}.{$endpoint_name}
+```
```

Comment:
Should we be validating the root_subject and endpoint_name the same as service name (`A-Z, a-z, 0-9, dash, underscore`)? Or maybe a different validation, but still a validation?

Reply:
Hmm, possibly we should validate, but I think not the same way as service name, but rather the same way as any NATS subject - we may have subject delimiters or wildcards in `root_subject`.

Reply:
the subscription will fail if they do it wrong, so no validation here.

Reply:
Note that it should be possible to do:

`accounts.*` for a root subject, and have endpoints that are:
`add`
`delete`


In that case the wildcard in the root subject is related to an account ID. - The info endpoint can take a number of other arguments. This is still a valid service. Also the endpoint could have a wildcard as well.